src/components/Links.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+// props = {}
+function Links({ github, linkedin }) {
+  return (
+    <div>
+      <h3>Links</h3>
+      <a href={github}> {github}</a>
+      <a href={linkedin}>{linkedin}</a>
+    </div>
+  );
+}
```

Comment:
These are the same thing:
```suggestion
function Links(props) {
  return (
    <div>
      <h3>Links</h3>
      <a href={props.github}> {props.github}</a>
      <a href={props.linkedin}>{props.linkedin}</a>
    </div>
  );
}
```